docs/product/components/toggle-switch.html

```diff
@@ -0,0 +1,249 @@
+description: Toggle switches prompt users to choose between mutually exclusive options. They are based on real-life on/off switches.
+---
+
+<div class="s-toast js-notice-toast" aria-hidden="true">
+    <aside class="s-notice py16 s-notice__success" role="status" aria-hidden="false">
+        <p class="m0">Setting changed to <strong id="js-value-placeholder"></strong>.</p>
+    </aside>
+</div>
+
+<section class="stacks-section">
+    <p class="stacks-copy">Toggles switches are a quick way to view and switch between mutually exclusive options. Use toggles when your intent is to instantly turn something on or off or change its frequency. If a real-life on/off switch would work for the action, a toggle is probably the best component to use.</p>
+    {% header "h2", "Classes" %}
+    <div class="overflow-x-auto">
+        <table class="wmn4 s-table s-table__bx-simple">
+            <thead>
+                <tr>
+                    <th class="s-table--cell4" scope="col">Class</th>
+                    <th scope="col">Description</th>
+                </tr>
+            </thead>
+            <tbody class="fs-caption">
+                <tr>
+                    <th scope="row"><code class="stacks-code">s-toggle-switch</code></th>
+                    <td>Base parent container for toggle switches</td>
+                </tr>
+                <tr>
+                    <th scope="row"><code class="stacks-code">s-toggle-switch--label-off</code></th>
+                    <td>Apply to the “Off” or “None” option so it’s highlighted state isn’t lit up green.</td>
+                </tr>
+            </tbody>
+        </table>
+    </div>
+</section>
+<section class="stacks-section">
+    {% header "h2", "Examples" %}
+    {% header "h3", "Base style" %}
+    <p class="stacks-copy">Basic toggles switches allow you to choose between two opposing states.</p>
+    <div class="stacks-preview">
+    {% highlight html %}
+<div class="grid ai-center">
+    <div class="s-toggle-switch mr8">
+        <input type="radio" name="group" id="one" checked>
+        <label for="one">On</label>
+        <input type="radio" name="group" id="two">
+        <label for="two" class="s-toggle-switch--label-off">Off</label>
+    </div>
+    <label class="s-label">…</label>
+</div>
+    {% endhighlight %}
+        <div class="stacks-preview--example">
+            <div class="grid ai-center">
+                <div class="s-toggle-switch mr8">
+                    <input type="radio" name="example1" id="example1-on" checked value="On">
+                    <label for="example1-on">On</label>
+                    <input type="radio" name="example1" id="example1-off" value="Off">
+                    <label for="example1-off" class="s-toggle-switch--label-off">Off</label>
+                </div>
+                <label class="s-label">Label</label>
+            </div>
+        </div>
+    </div>
+</section>
+<section class="stacks-section">
+    {% header "h3", "3+ options" %}
```

Comment:
```suggestion
    {% header "h3", "3 or more options" %}
```

Way better slug this way.